tasks.py

```diff
@@ -50,6 +50,11 @@
 RENDER_MATH_DIR = PLUGINS_DIR / "render_math"
 ZIP_FILE_RM = Path("render-math.zip")
 
+PELICAN_CITE_ZIP_URL = ("https://github.com/VorpalBlade/pelican-cite/archive/refs/heads/master.zip")
+PELICAN_CITE_DIR = PLUGINS_DIR / "pelican-cite"
+ZIP_FILE_PC = Path("pelican-cite.zip")
```

Comment:
We cannot keep adding new variables and new methods each time we need to add more plugins.

Create a [TypedDict](https://typing.python.org/en/latest/spec/typeddict.html) called PLUGINS, that has three variables, download_url, plugin_dir, plugin_filename. In method download_plugins and clean_plugins, we iterate over each element and do the download and cleaning.